### ISSUE TITLE
[BUG] Repeater do not load all its values

### ISSUE THREAD

**Issue by npanau** [bolt/bolt#5668]
I have a contenttype with a repeater field type. Its stores my 11 eleven blocs, but if I re-edit the content, it do not list them all back, so if I store, I loose some blocs.

Reproduction
------------
This repeater has 4 fields (bool, text, text, select), and a limit to 11 bloc.
The first time I created 11 blocs, evertything is fine : they are saved in database, and I can get these 11 blocs in twig.
But if I edit the content again (or simply refresh the page), Bolt do not load all the stored blocs ; it gets only the 7th first, and only the first field of the 8th.

I checked in the profiler, and i did see this behaviour ; seven times : 
```
 SELECT field_value.* FROM bolt_field_value field_value WHERE field_value.id IN(:id)

Parameters: {":id":["561","562","563","564"]} 
```
(with inc id obviously :))
and one time : 
```
SELECT field_value.* FROM bolt_field_value field_value WHERE field_value.id IN(:id)

Parameters: {":id":["574"]} 
```
These ids are the one of the repeater.

Note : I splitted my repeater in 2 (6 and 5 limit), and everything is OK. Seems that if there are too many block, it can't call them back, but can store them...

Details
-------
 - Relevant Bolt Version: [ 3.0.11 ]
 - Install type: [ Composer install ]
 - PHP version: [ 5.6 | 7.0 ]
 - Used web server: [ Apache ] version [ 2.4 ]

**Comment by rossriley** [comment]
Hi @npanau this was one that we came across and fixed here #5581 but it's not yet in a released version of Bolt.

It's a mysql setting that you can adjust too yourself though so if you are able to you can set group_concat_max_len either in global my.cnf or by running a `SET @@global` query.

**Comment by bobdenotter** [comment]
The fix will be in Bolt 3.0.12, which will be out _this week_. So if you can wait a little bit longer, the update will help you out.

**Comment by npanau** [comment]
Oh! I've searched but found nothing.
My purpose was mainly to help improve this great app :)
I close since it's resolved!
Thx & keep up the good work :)
